### PR TITLE
Updating sbrscrape to fix fanduel odds source and adding tensorflow

### LINKED ISSUE #200: TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
When I run the script it outputs odds but fails near the end with NoneType issue.  Any ideas?

python3 main.py -A -odds=fanduel
2023-04-21 08:55:15.762270: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2023-04-21 08:55:15.801673: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2023-04-21 08:55:15.801965: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-04-21 08:55:16.289200: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2023-04-21 08:55:16.904949: I tensorflow/compiler/xla/stream_executor/cuda/cuda_gpu_executor.cc:996] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero. See more at https://github.com/torvalds/linux/blob/v6.0/Documentation/ABI/testing/sysfs-bus-pci#L344-L355
2023-04-21 08:55:16.905268: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1956] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
------------------fanduel odds data------------------
Boston Celtics (None) @ Atlanta Hawks (None)
Cleveland Cavaliers (None) @ New York Knicks (None)
Denver Nuggets (None) @ Minnesota Timberwolves (None)
---------------XGBoost Model Predictions---------------
Atlanta Hawks vs Boston Celtics (57.7%): OVER None (92.4%)
New York Knicks (55.6%) vs Cleveland Cavaliers: OVER None (92.8%)
Minnesota Timberwolves vs Denver Nuggets (55.0%): OVER None (96.5%)
--------------------Expected Value---------------------
Atlanta Hawks EV: 0
Boston Celtics EV: 0
New York Knicks EV: 0
Cleveland Cavaliers EV: 0
Minnesota Timberwolves EV: 0
Denver Nuggets EV: 0
-------------------------------------------------------
------------Neural Network Model Predictions-----------
1/1 [==============================] - 0s 71ms/step
1/1 [==============================] - 0s 12ms/step
1/1 [==============================] - 0s 14ms/step
1/1 [==============================] - 0s 34ms/step
1/1 [==============================] - 0s 11ms/step
1/1 [==============================] - 0s 11ms/step
Atlanta Hawks vs Boston Celtics (64.8%): UNDER None (nan%)
New York Knicks (58.2%) vs Cleveland Cavaliers: UNDER None (nan%)
Minnesota Timberwolves (51.1%) vs Denver Nuggets: UNDER None (nan%)
--------------------Expected Value---------------------
Traceback (most recent call last):
  File "/root/nba/main.py", line 131, in <module>
    main()
  File "/root/nba/main.py", line 120, in main
    NN_Runner.nn_runner(data, todays_games_uo, frame_ml, games, home_team_odds, away_team_odds)
  File "/root/nba/src/Predict/NN_Runner.py", line 66, in nn_runner
    ev_home = float(Expected_Value.expected_value(ml_predictions_array[count][0][1], int(home_team_odds[count])))
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'

### PR DESCRIPTION
**Pull request by chriseling**
Updating sbrscrape to fix fanduel odds data source and adding tensorflow to requirements.  

Closes #200

### CODE REVIEW

**Comment by kyleskom** [comment]
I don't think fan duel was having any issues. I think user just ran to early in the day.

**Comment by kyleskom** [comment]
Also tensorflow 2.11.0 should be the version. Another PR was open for that I just merged

**Comment by kyleskom** [comment]
Can you remove the tensorflow pip install. I think we have that covered now. Will test the other.

File: requirements.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ numpy==1.24.2
 scikit-learn==1.2.1
 keras==2.11.0
 xgboost==1.7.4
-sbrscrape==0.0.6
+sbrscrape==0.0.8
 Flask==2.2.3
 openpyxl==3.1.1
+tensorflow==2.12.0
```
Comment (kyleskom) [inline requirements.txt:12]: Lets remove this

**Comment by onlyadamscott** [comment]
> 

Also, sbrscrape is pulling accurate odds from all books out of the box. Not sure what the issue you are seeing is.

**Comment by onlyadamscott** [comment]
> > 
> 
> Also, sbrscrape is pulling accurate odds from all books out of the box. Not sure what the issue you are seeing is.

I've tested multiple times today

File: requirements.txt (hunk 1 above)
Comment (chriseling) [reply requirements.txt]: sounds good

**Review by kyleskom — APPROVED** [review]

**Pull request merged**